site/content/docs/howto/debugging.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Debugging Pinniped 
+description: Debugging `pinniped` configuration errors.
+description: 
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Debugging
+    weight: 900
+    parent: howtos
+---
+
+## Debugging on the Client 
+
+The `PINNIPED_DEBUG=true` environment variable can be set to enable additional CLI logging.
+
+## Debugging on the Server
+
+To adjust the log level of the server side components of Pinniped (such as the Supervisor, Concierge,
+Impersonation Proxy, etc) edit the `log_level` configuration in your local copy of the Pinniped GitHub 
+repository. Open the `deploy/supervisor/values.yaml` or `deploy/concierge/values.yaml` file and edit: 
+
+```yaml
+log_level: "info|debug|trace|all"
+```
+Then apply your configuration.
```

Comment:
Log level only applies to the Supervisor and Concierge.

These instructions assume that you are going to use ytt to deploy, but we actually document several ways to install the Supervisor and the Concierge (ytt or use yaml attached to our GitHub releases). This should describe both, or describe where to edit the value live on the cluster. Note that if you edit the value live on the cluster then you also need to restart the pods.

Reply:
Makes sense, the user should know what value in the configmap is being updated by this change.

Reply:
Agree, updating.